compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -16,13 +16,30 @@
 
 #include "RecordMinMax.h"
 #include "CircleExpContract.h"
+#include "HDF5Importer.h"
 
 #include <luci/Importer.h>
 #include <luci/CircleExporter.h>
 
 #include <fstream>
 #include <stdexcept>
 
+using Shape = luci_interpreter::Shape;
+using DataType = luci_interpreter::DataType;
+
+namespace
+{
+
+template <typename NodeT> size_t getTensorSize(const NodeT *node)
```

Comment:
(optional) maybe add a brief about this will return Tensor size in bytes ?

Reply:
Added a comment. Thanks.